models/issues/issue.go

```diff
@@ -596,6 +596,8 @@ func IsUserParticipantsOfIssue(ctx context.Context, user *user_model.User, issue
 type DependencyInfo struct {
```

Comment:
document: this type is used in func X,Y,Z as xorm intermediet type to retrive info from joined tables ...

Reply:
Thanks!